Resources/ServerInfo/Guidebook/DeltaV/Rules/RoleRules/C3_Antags.xml

```diff
@@ -1,9 +1,12 @@
 <Document>
   # Rule C3: Antagonist Guidelines
+[color=#ffff00]Being an antagonist does not allow you to stop playing a character. Determine how your character would react to being given these objectives, and work through it appropriately.[/color]
+
 Through engaging in antagonistic activity, you should seek to make the round more engaging and fun as the primary driver of the narrative of a round. Succeeding as an antagonist should not be your goal, but rather telling the most interesting story for everyone involved. 
   - [color=#ff0000]Antagonists are free to complete their objectives through committing proportional damage. Through roleplay, damage becomes more proportional.[/color] Example: Holding the singulo hostage for a theft objective is acceptable, while simply singuloosing and using that chaos is unacceptable. 
     - Killing players unrelated to your immediate objective in a manner that results in their round removal should be avoided. Crew that do not make an attempt at self preservation, or engage in valid-hunting, are exempt from this. 
     - If you are concerned as to whether or not what you're about to do is allowed, feel free to AHelp and ask an admin for clarification. [color=#ff0000]Lack of administrator response does not constitute approval.[/color]
+  - While antagonists are [color=#ffff00]not required[/color] to complete their objectives, they are still encouraged to act as an antagonist in the round while roleplaying. 
```

Comment:
I do feel like this slightly increases the contradiction with the metafriending rules on how it's metafriending to ignore a kill objective if your characters are friends (from a previous round). 

https://github.com/DeltaV-Station/Delta-v/blob/648396c6635d957f21fab291ce282ae9df3487ae/Resources/ServerInfo/Guidebook/DeltaV/Rules/GameRules/2_Metagaming.xml#L119

Reply:
My bad, this was discussed in the discord as not really being a big deal, nor enforceable. It should be removed.

Reply:
Good because TBH I pretended like that rule didn't exist already. :sunglasses:

(I barely ever have antags turned on anyway so this rebellion against the rules isn't quite as cool as it sounds.)